django/contrib/gis/db/models/fields.py

```diff
@@ -405,6 +405,11 @@ class RasterField(BaseSpatialField):
     description = _("Raster Field")
     geom_type = 'RASTER'
 
+    def __init__(self, **kwargs):
```

Comment:
I think we need to accept *args here too (or at least at `verbose_name=None`. There's a convention to pass `verbose_name` as a positional arg: https://docs.djangoproject.com/en/1.8/topics/db/models/#verbose-field-names
Add a test for that too?

Claude, is this approach fine with you?